rmf_task/include/rmf_task/Estimate.hpp

```diff
@@ -28,30 +28,34 @@
 namespace rmf_task {
 
 //==============================================================================
-/// Estimates for requests
+/// A class to store the state of an AGV after finishing the request and the
+/// time that the AGV should wait till before executing the request.
```

Comment:
Super minor nitpick: I might suggest flipping this sentence so it mentions the start time before it mentions the finish state estimation. I think mentioning these steps in chronological order will paint a more clear picture for most human brains.

Reply:
Done https://github.com/open-rmf/rmf_task/pull/28/commits/c75294eae225ef8cd38657b95a565de42edf11e6